cli/src/bin/phylum.rs

```diff
@@ -93,6 +105,20 @@ async fn handle_commands() -> CommandResult {
         )
     })?;
 
+    // `timeout` and `ignore_certs` are arguments to the API factory.
```

Comment:
This comment seems somewhat strange to me, since it documents how variables are used which I can usually just check with my IDE's functionality.

If the goal is to document that we're initializing everything required to create the API, that should be what is stated in the comment. This also comes with the advantage of being more generic and thus being more robust to changes.

Reply:
Indeed, those two variables are used only a bit later but I didn't find an appropriate spot for them. I will change the comment, but the best thing imo would be to refactor the data dependencies between `Config` and `PhylumApi` in such a way that these bindings can be removed altogether. I am afraid it might not be super trivial, though.